src/transport/raw/TCP.cpp

```diff
@@ -353,7 +343,7 @@ CHIP_ERROR TCPBase::ProcessSingleMessage(const PeerAddress & peerAddress, Active
         message = System::PacketBufferHandle::New(messageSize, 0);
         if (message.IsNull())
         {
-            return CHIP_ERROR_NO_MEMORY;
+            return INET_ERROR_NO_MEMORY;
         }
         CHIP_ERROR err = state->mReceived->Read(message->Start(), messageSize);
```

Comment:
This ends up being possibly-returned via `ReturnErrorOnFailure`, but it's not an INET_ERROR.